Gulpfile.mjs

```diff
@@ -351,6 +351,10 @@ function buildRollup(packages, targetBrowsers) {
                   "regenerate-unicode-properties"
                 ) + "/**/*.js",
               ],
+              // Never delegate to the native require()
+              ignoreDynamicRequires: true,
+              // Align with the Node.js behavior
+              defaultIsModuleExports: true,
```

Comment:
This makes it possible to remove a custom plugin from our Babel config, by moving the logic to Rollup itself.